wheel_cfg:
  closedloopRamp: 1
  slot0: {
    kP: 0.1,
    kI: 0.002,
    kD: 5.0,
    kF: 0.035,
    maxIntegralAccumulator: 16000
  }

bscrew_cfg:
  drivePosition: 0
  depositPosition: -2000000
  digPosition: -4000000
  miliseconds: 1000

  motionCruiseVelocity: 25000
  motionAcceleration: 100000
  motionCurveStrength: 0
  clearPositionOnLimitF: true

  slot0: {
    kP: 0.5, #0.5
    kI: 0.00000 # 0
    # maxIntegralAccumulator: 50000
  }

linact_cfg:
  drivePosition: -16 # 0
  depositPosition: 0 # 10
  digPosition: -30 # -60
  miliseconds: 1000

  motionCruiseVelocity: 4
  motionAcceleration: 10
  motionCurveStrength: 0

  slot0: {
    kP: 42, #45
    kI: 0.001 # 0.0001
  }

